blog/2023-05-04-announcing-realtime-reporting-for-stryker.mdx

```diff
@@ -0,0 +1,36 @@
+---
+slug: announcing-realtime-reporting-for-stryker
+title: 'Announcing Realtime Reporting for Stryker'
+authors: xander-vedder
+tags: [stryker-js, stryker-net, typescript, C#]
+---
+
+# Realtime Reporting
+
+Wait no longer to see the results of your testing! With this new addition to Stryker, you can receive real-time feedback as your code is being tested, enabling you to identify and address issues as soon as they occur!
+
+<!-- truncate -->
+
+## The problem
+
+Currently developers must wait until the end of the mutation testing cycle to receive feedback on the results of mutation testing. This delay in feedback can result in a slower and less efficient testing process, as developers may need to spend additional time debugging and fixing issues that could have been identified earlier in the process.
+
+## The solution
+
+The solution to the delayed feedback in mutation testing is real-time reporting. With this approach, the report is updated immediately as the testing process is ongoing, providing developers with immediate feedback. This means that developers can identify and address issues in their code more quickly, rather than waiting until testing is complete to see the results. By enabling developers to access feedback in realtime, realtime reporting significantly improves the efficiency and effectiveness of the testing process.
```

Comment:
```suggestion
With real-time reporting the html report is updated whenever a new mutant has been tested, providing developers with immediate feedback. Developers can identify and address issues in their code earlier, because they don't have to wait until testing is finished to see results. 
```